docs/modules/ROOT/pages/explanations/connectiondetails.adoc

```diff
@@ -0,0 +1,227 @@
+= Connection Details
+
+This page will try to document the various layers of connection details and how to handle them.
+
+Please read https://docs.crossplane.io/knowledge-base/guides/connection-details/[Crossplane's docs about this topic first].
+It explains the various connection details very well.
+The following page assumes you've read and understand.
+
+== Composite Resource Connection Details
+
+These are the most straight forward kind of connection details to handle.
+In terms of go code they are specified by `xfnv1alpha1.ExplicitConnectionDetail`.
+As the name suggests, they are explicit and take a name and a value.
+
+[source,golang]
+----
+iof.Desired.PutCompositeConnectionDetail(ctx, xfnv1alpha1.ExplicitConnectionDetail{
+	Name:  "ENDPOINT",
+	Value: bucket.Status.Endpoint,
+})
+----
+
+== Managed Resource Connection Details
+
+Each manged resource can optionally expose its own connection details.
+Some examples:
+
+* Exoscale IAM
+* Minio User
+* Helm Release
+
+Let's have a look at an easy example and a more complicated one.
+
+=== Simple Example
+
+We use the minio User managed resource in this example.
+The User can expose two connection details: `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`.
+It's important that we set the `WriteConnectionSecretToReference`.
+
+[source,golang]
+----
+user := &miniov1.User{
+  ...
+	Spec: miniov1.UserSpec{
+    Spec: ...
+		ResourceSpec: xpv1.ResourceSpec{
+			WriteConnectionSecretToReference: &xpv1.SecretReference{
+				Name:      "uniqname", <1>
+				Namespace: "secretnamespace", <2>
+			},
+		},
+	},
+}
+----
+<1> Unique name for this secret, usually the composite name + some suffix
+<2> The namespace where the secret should be saved, usually set to `writeConnectionSecretsToNamespace`.
```

Comment:
`writeConnectionSecretsToNamespace` - we can specify that this is from composition.